src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/helper/DocumentHelper.java

```diff
@@ -222,6 +230,10 @@ public String formatAddressForLetterPrinting(Map<String, Object> address) {
             .collect(Collectors.joining("\n"));
     }
 
+    public BulkPrintDocument getCaseDocumentAsBulkPrintDocument(CaseDocument caseDocument) {
```

Comment:
we already have this method `getCaseDocumentAsBulkPrintDocument`

Reply:
couldn't find it - GitHub link?